File: Assignment_11/question4.c
///////////////////////////////////////////////////////////////////////////////////////
//
//  File Name   : question4.c
//  Description : Display # number * pattern.
//
//  Problem     : Accept number from user and display pattern: # 1 * # 2 * # 3 * # 4 *
//
//  Author      : Asawari Jadhav
//  Date        : 20/05/2025
//
//////////////////////////////////////////////////////////////////////////////////////

#include <stdio.h>

void Pattern(int iNo)
{
  int iCnt = 0;
  for (iCnt = 1; iCnt <= iNo; iCnt++)
  {
    printf("#\t%d\t*\t", iCnt);
  }
}

int main()
{
  int iValue = 0;

  printf("Enter number of elements\n");
  scanf("%d", &iValue);

  Pattern(iValue);

  return 0;
}

File: Assignment_50/question4.java
//////////////////////////////////////////////////////////////////////////////////////////////
//
//  File Name     : question4.java
//  Description   : Accept number of rows and columns from user and display solid star pattern
//  Problem       : Display * in all positions of the matrix
//
//  Author        : Asawari Jadhav
//
//  Date          : 21-08-2025            
//
//////////////////////////////////////////////////////////////////////////////////////////////

import java.util.Scanner;

class Pattern 
{
    public void DisplayPattern(int iRow, int iCol) 
    {
        for (int i = 0; i < iRow; i++) 
        {
            for (int j = 0; j < iCol; j++)
            {
                System.out.print("* ");
            }
                
            System.out.println();
        }
    }
}

public class question4 
{
    public static void main(String args[]) 
    {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter number of rows: ");
        int iRow = sc.nextInt();
        System.out.print("Enter number of columns: ");
        int iCol = sc.nextInt();

        Pattern pobj = new Pattern();
        pobj.DisplayPattern(iRow, iCol);

        
    }
}
